front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -5,26 +5,9 @@ import styled from 'styled-components';
 
 import useInstanceId from '../../hooks/useInstanceId';
 import { isRtl } from '../../utils/styleUtils';
-import Box, { BoxMarginProps, BoxPaddingProps, BoxWidthProps } from '../Box';
+import Box, { BoxProps } from '../Box';
 import Icon from '../Icon';
-import Title, { Variant, FontSize, FontWeight } from '../Title';
-
-type CollapsibleContainerProps = {
-  children: ReactNode;
-  title: ReactNode;
-  isOpenByDefault?: boolean;
-  useRegionRole?: boolean;
-} & BoxMarginProps &
-  BoxWidthProps &
-  BoxPaddingProps &
-  TitleProps;
-
-type TitleProps = {
-  titleVariant: Variant;
-  titleFontSize?: FontSize;
-  titlePadding?: string;
-  titleFontWeight?: FontWeight;
-};
```

Comment:
(Personal preference: have props right above the component, where you need to check them.)